__tests__/setup.test.js

```diff
@@ -30,6 +30,9 @@ describe('A test suite example using enzyme', function() {
     expect(render(<ReviewService />).text()).toEqual('Reviews');
   });
 
+});
+
+describe('A test suite for Review Container', () => {
```

Comment:
this line could read 'Review container'  because when the test is run we know it is a test suite